src/nomad_simulations/schema_packages/model_method.py

```diff
@@ -838,7 +838,7 @@ class ExcitedStateMethodology(ModelMethodElectronic):
     A base section used to define the parameters typical of excited-state calculations. "ExcitedStateMethodology"
     mainly refers to methodologies which consider many-body effects as a perturbation of the original
     DFT Hamiltonian. These are: GW, TDDFT, BSE.
-    """
+    """  # Note: we don't really talk about Hamiltonians in DFT: their physics is accommodated in the functional itself
```

Comment:
what?

Reply:
I'd change the phrasing from "DFT Hamiltonian" to "density functional", since the former is not really used.

Overall, I associate "Hamiltonian" more so with operators acting upon vectors / quantum states.
DFT, meanwhile, rewrites the Hamiltonian to be a functional of another function, typically the external potential $v\left(\vec{r}\right)$. To rephrase it in terms of the electronic density function $n\left(\vec{r}\right)$, it uses a Legendre transform. Therefore, I'd consider it a different mathematical object.

Reply:
> Therefore, I'd consider it a different mathematical object.

It is not.